core/src/main/java/nl/nn/adapterframework/doc/model/ElementType.java

```diff
@@ -35,6 +37,15 @@
 	private @Getter String simpleName;
 	private @Getter Map<String, FrankElement> members;
 	private @Getter boolean fromJavaInterface;
+	
+	/**
+	 * For each syntax 1 name in this set, an &lt;xs:choice&gt; of &lt;xs:element&gt;
+	 * is created in the XSD. The elements in a group correspond to the Java
+	 * classes that implement the Java interface represented by this {@link ElementChild}.
+	 * For each syntax 1 name a different group is needed because the syntax 2
+	 * names of the elements are different.
+	 */
+	private @Getter Set<String> configChildSyntax1Names = new TreeSet<>();
```

Comment:
Als je in plaats van een TreeSet een LinkedHashSet gebruikt, blijft de volgorde van toevoegen behouden. Dan komt Sender voor ErrorSender, en dat is prettig.